### PR TITLE
Adding Example for Cooldowns. (No Code Changes)

### PR DESCRIPTION
**Pull request by xCykrix**
Adds a simple cooldown example to documentation for people who don't know how. I've seen a few people ask how to do this in discord.js

### CODE REVIEW

File: docs/examples/cooldowns.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+// from Discord _after_ ready is emitted
+client.on('ready', () => {
+  console.log('I am ready!');
+});
+
+// Cooldown User Object (Empty Array)
+var cooldownUsers = [];
+
+// Cooldown Check Function
+const checkCooldown = ((userId) => {
```
Comment (iCrawl) [inline docs/examples/cooldowns.js]: This can be simplified and also use `includes()` instead of `indexOf`

File: docs/examples/cooldowns.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+const token = 'your bot token here';
+
+// The ready event is vital, it means that your bot will only start reacting to information
+// from Discord _after_ ready is emitted
+client.on('ready', () => {
+  console.log('I am ready!');
+});
+
+// Cooldown User Object (Empty Array)
+var cooldownUsers = [];
```
Comment (iCrawl) [inline docs/examples/cooldowns.js]: Also this could be `const`

File: docs/examples/cooldowns.js (hunk 1 above)
Comment (devsnek) [inline docs/examples/cooldowns.js]: this entire function is useless

**Comment by devsnek** [comment]
i would use a `Set` for the cooldown list, because you can just do `Set#add` and `Set#delete`

**Comment by devsnek** [comment]
```js
const Discord = require('discord.js');
const client = new Discord.Client();

const cooldowns = new Set();

client.on('ready', () => {
  console.log('READY', client.user.tag, client.user.id);
});

function cooldown(user, time) {
  cooldowns.add(user);
  setTimeout(() => cooldowns.delete(user), time * 1000);
}

client.on('message', (message) => {
  if (message.author.bot || cooldowns.has(message.author)) return;
  if (message.content === '!ping') {
    message.channel.send('pong!');
    cooldown(message.author, 5);
  }
});
```

File: docs/examples/cooldowns.js (hunk 1 above)
Comment (eaaliprantis) [inline docs/examples/cooldowns.js]: I don't really think the entire function is useless.  Some people don't really know how to create cooldowns for their commands when they are learning how to code.

Comment (appellation) [inline docs/examples/cooldowns.js]: This is an example not a tutorial.

**Comment by gainagames** [comment]
How would you change console.log(`${user.tag} (${user.id}) is cooling down for ${time} seconds`); to send that in the channel?

**Comment by eaaliprantis** [comment]
message.send(<insert message here>)

**Comment by BigBrainAFK** [comment]
I dont think timeouts are necessary. Just a collection or smth with the user and the unix timestamp of when the cooldown ends... then just check if the curremt timestamp is greater or equal on every command the user tries to use

**Pull request closed without merging**